src/modules/poweraccent/PowerAccent.Core/Languages.cs

```diff
@@ -0,0 +1,501 @@
+﻿// Copyright (c) Microsoft Corporation
+// The Microsoft Corporation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace PowerAccent.Core
+{
+    using System;
+    using PowerToys.PowerAccentKeyboardService;
+
+    public enum Language
+    {
+        ALL,
+        CUR,
+        CZ,
+        DE,
+        FR,
+        HU,
+        IS,
+        IT,
+        MI,
+        PI,
+        PL,
+        RO,
+        SK,
+        SP,
+        TK,
+    }
+
+    internal class Languages
+    {
+        public static char[] GetDefaultLetterKey(LetterKey letter, Language lang)
+        {
+            switch (lang)
+            {
+                case Language.ALL: return GetDefaultLetterKeyALL(letter); // ALL
+                case Language.CUR: return GetDefaultLetterKeyCUR(letter); // Currency
+                case Language.CZ: return GetDefaultLetterKeyCZ(letter); // Czech
+                case Language.DE: return GetDefaultLetterKeyDE(letter); // German
+                case Language.FR: return GetDefaultLetterKeyFR(letter); // French
+                case Language.HU: return GetDefaultLetterKeyHU(letter); // Hungarian
+                case Language.IS: return GetDefaultLetterKeyIS(letter); // Iceland
+                case Language.IT: return GetDefaultLetterKeyIT(letter); // Italian
+                case Language.MI: return GetDefaultLetterKeyMI(letter); // Maori
+                case Language.PI: return GetDefaultLetterKeyPI(letter); // Pinyin
+                case Language.PL: return GetDefaultLetterKeyPL(letter); // Polish
+                case Language.RO: return GetDefaultLetterKeyRO(letter); // Romanian
+                case Language.SK: return GetDefaultLetterKeySK(letter); // Slovak
+                case Language.SP: return GetDefaultLetterKeySP(letter); // Spain
+                case Language.TK: return GetDefaultLetterKeyTK(letter); // Turkish
+            }
+
+            throw new ArgumentException("The language {0} is not know in this context", lang.ToString());
+        }
+
+        // All
+        private static char[] GetDefaultLetterKeyALL(LetterKey letter)
+        {
+            switch (letter)
+            {
+                case LetterKey.VK_0:
+                    return new char[] { '₀', '⁰' };
+                case LetterKey.VK_1:
+                    return new char[] { '₁', '¹' };
```

Comment:
On German keyboard we have often used nunbers like ½, ⅔, ¾, ⅘ on the numbers. (At least on my mobile phone.)